examples/text_labels_and_annotations/legend_demo.py

```diff
@@ -165,7 +163,7 @@ def create_artists(self, legend, orig_handle,
 
 x = np.linspace(0, 5, 100)
 
-fig, ax = plt.subplots()
+fig, ax = plt.subplots(constrained_layout=True)
```

Comment:
There doesn't seem to be any need for constrained layout in this case.

Reply:
I just leave it on by default :wink: (but removed, because you are correct, it only fills the plot out a bit more, but doesn't correct any actual bad formatting...